### ISSUE TITLE
Script fails when there's no internet connectivity to nsp

### ISSUE THREAD

**Issue by marcellodesales** [nodesecurity/nsp#66]
The failure occurs when trying to run the report when there's no connectivity with the NSP server.

```js
/home/mdesales/dev/github-intuit/servicesplatform-nodejs/sp-quality/node_modules/gulp-nsp/index.js:27
    if (params.stopOnError === false || data.length === 0) {
                                            ^

TypeError: Cannot read property 'length' of undefined
    at /home/mdesales/dev/github-intuit/servicesplatform-nodejs/sp-quality/node_modules/gulp-nsp/index.js:27:45
    at /home/mdesales/dev/github-intuit/servicesplatform-nodejs/sp-quality/node_modules/gulp-nsp/node_modules/nsp/lib/check.js:207:16
    at /home/mdesales/dev/github-intuit/servicesplatform-nodejs/sp-quality/node_modules/gulp-nsp/node_modules/nsp/node_modules/wreck/lib/index.js:467:20
    at finish (/home/mdesales/dev/github-intuit/servicesplatform-nodejs/sp-quality/node_modules/gulp-nsp/node_modules/nsp/node_modules/wreck/lib/index.js:144:20)
    at wrapped (/home/mdesales/dev/github-intuit/servicesplatform-nodejs/sp-quality/node_modules/gulp-nsp/node_modules/nsp/node_modules/hoek/lib/index.js:866:20)
    at ClientRequest.onError (/home/mdesales/dev/github-intuit/servicesplatform-nodejs/sp-quality/node_modules/gulp-nsp/node_modules/nsp/node_modules/wreck/lib/index.js:153:16)
    at ClientRequest.g (events.js:260:16)
    at emitOne (events.js:77:13)
    at ClientRequest.emit (events.js:169:7)
    at TLSSocket.socketErrorListener (_http_client.js:259:9)
    at emitOne (events.js:77:13)
    at TLSSocket.emit (events.js:169:7)
    at connectErrorNT (net.js:996:8)
    at doNTCallback2 (node.js:441:9)
    at process._tickDomainCallback (node.js:396:17)
npm info sp-quality@1.0.0-10 Failed to exec test script
npm ERR! Test failed.  See above for more details.

```

**Comment by jlamendo** [comment]
Thanks for reporting this issue marcello.
This was resolved with commit [#f23f232](https://github.com/nodesecurity/gulp-nsp/commit/f23f23268c9d621aeeb87aa4073f13fd446122da) to gulp-nsp. Closing the issue for now.

**Comment by marcellodesales** [comment]
@jlamendo thank you!
